src/app/attendance/actions/updateAttendance.ts

```diff
@@ -0,0 +1,26 @@
+"use server";
+
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+import { AttendanceFormValues } from "../components/AttendanceFormBuilder";
+import { redirect } from "next/navigation";
+import { Routes } from "@/lib/routes";
+
+interface Attendance extends AttendanceFormValues  {
+  id: string | string[];
+};
+
+export const updateAttendance = async (attendance : Attendance) => {
```

Comment:
can we use `kebab case` naming convention here?

Reply:
Fixed